tests/integration/modules/test_grains.py

```diff
@@ -152,8 +152,7 @@ class GrainsAppendTestCase(ModuleCase):
     GRAIN_VAL = 'my-grain-val'
 
     def tearDown(self):
-        for item in self.run_function('grains.get', [self.GRAIN_KEY]):
-            self.run_function('grains.remove', [self.GRAIN_KEY, item])
```

Comment:
OK, but why is this necessary? I don't understand what's being fixed here. Is `grains.remove` not doing the right thing?

Reply:
In rare cases the `grains.get` function was returning a boolean value and the following stack trace was thrown:
```
Traceback (most recent call last):
  File "c:\testing\tests\integration\modules\test_grains.py", line 155, in tearDown
    for item in self.run_function('grains.get', [self.GRAIN_KEY]):
TypeError: 'bool' object is not iterable
```
So rather than loop over the grains just to remove each item, we decided to just set it to an empty list.

Reply:
@cachedout Can you swing back around here when you get a moment?